entrypoint.sh

```diff
@@ -4,9 +4,19 @@ set -e
 zulip_organization_url=$1
 zulip_bot_email=$2
 zulip_bot_api_key=$3
-github_personal_access_token=$4
+github_token=$4
 delete_history=$5
 archive_branch=$6
+github_personal_access_token=$7
+
+github_personal_access_token=${github_personal_access_token:-NOT_SET}
+
+if [ $github_personal_access_token != "NOT_SET" ]; then
+    echo "'github_personal_access_token' input has been deprecated."
+    echo "Please use 'github_token' instead."
+    echo "For further information, please read the README.md in https://github.com/zulip/zulip-archive"
```

Comment:
The readme no longer has info about GH personal access token, and so I think we should link to this PR instead.